Make public displays feature accessible to users

ldodds opened the pull request:
- [x] Change URL paths for feature, moving files around
- [x] Move index to a controller under /schools and ensure only school users can access
- [x] Add Manage My School link
- [x] Add Manage Group link
- [x] Revise school page to indicate if feature isn't available
- [x] Revise school page to include more instructions
- [x] Add School group index page with instructions
- [x] Provide option to download links for whole school group

urbanwide commented on app/views/school_groups/digital_signage/index.html.erb:
```diff
@@ -0,0 +1,60 @@
+<div class="row">
+  <div class="col-12">
+    <%= render PromptComponent.new status: :neutral, icon: 'info-circle' do |c| %>
+      <% c.with_link { link_to 'Get in touch', 'mailto:hello@energysparks.uk' } %>
```
Do these bits of text need translating?

ldodds replied:
Yes, that's me rushing! :man_facepalming:

urbanwide commented on app/views/schools/digital_signage/index.html.erb:
```diff
@@ -0,0 +1,122 @@
+  <div class="col-12">
+    <%= render PromptComponent.new status: :neutral, icon: 'info-circle' do |c| %>
+      <% c.with_link { link_to 'Get in touch', 'mailto:hello@energysparks.uk' } %>
+      <% c.with_title { 'We are inviting schools to try out this new feature of Energy Sparks' } %>
```
Same here for i18n

urbanwide commented:
![image](https://github.com/user-attachments/assets/a9db2c78-0013-4a4e-9910-9161f31dac07)
If you cut and paste the displayed link, it doesn't work. Took me a while to work out it is because digital_signage has an underscore in it and not a hyphen as per the actual route.

urbanwide commented on app/controllers/pupils/digital_signage_controller.rb:
```diff
@@ -24,7 +24,7 @@ class PublicDisplaysController < ApplicationController
     end
 
     def index
```
I think this method can be removed, as I don't think there is a route for it anymore?

urbanwide reviewed (approved):
Looks good - a couple of questions, main one about the url structure.

ldodds commented:
Thanks for review @urbanwide , have added translations, removed index method, fixed the in-page link, and tweaked the URL structure. Agree it was inconsistent and likely to cause confusion!

Did you want to look again before I merge?

urbanwide reviewed (approved):
Looks good 👍🏻
